### ISSUE TITLE
Persist syntax style of opened document when creating a new one

### ISSUE THREAD

**Issue by nuno-agostinho** [coteditor/CotEditor#1165]
If I'm working on a Python script and open a new document, it would be nice if CotEditor automatically set the syntax to Python automatically.

**Comment by 1024jp** [comment]
You can set the default syntax style in Preferences > Format pane.

<img width="682" alt="Screen Shot 2021-04-06 at 22 01 20" src="https://user-images.githubusercontent.com/1165044/113714760-b4eccf00-9723-11eb-938c-fcd1aba6fdb9.png">

**Comment by nuno-agostinho** [comment]
Hey @1024jp, I know I can set the default, but I usually work with different programming languages, so I have to change the syntax all time. What I am asking is for the "Default syntax style" to have the option of "Last syntax in use", for instance.

**Comment by 1024jp** [comment]
Aha, I see. Thank you for the explanation. Hmm, then, let me think whether I implement or not.

**Comment by 1024jp** [comment]
I recently considered this feature again.
I have imagined different use cases and how I can concretely implement it, but I come to the conclusion that this feature is not as convenient as it initially sounds.

CotEditor is already designed to be used for different document types.
Even if the user opted in for this feature, it would be unintended in many situations for new documents to be affected by the last document opened, including documents that were opened unexpectedly.

Therefore, I decided not to implement this feature.
Sorry for that.
